src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetVerbCommand.cs

```diff
@@ -71,7 +71,9 @@ protected override void ProcessRecord()
 
                         VerbInfo verb = new VerbInfo();
                         verb.Verb = field.Name;
+                        verb.AliasPrefix = VerbAliasPrefixes.GetVerbAliasPrefix(field.Name);
                         verb.Group = groupName;
+                        verb.Description = VerbDescriptions.Get(field.Name);
```

Comment:
Above we use `VerbAliasPrefixes.GetVerbAliasPrefix`. I believe we should use the same pattern and use `VerbDescriptions.GetVerbDescription`.